libs/api/ebpf_api.cpp

```diff
@@ -2314,7 +2313,7 @@ CATCH_NO_MEMORY_EBPF_RESULT
 
 static ebpf_result_t
 _initialize_ebpf_object_from_file(
```

Comment:
nit: should we rename this function to remove `from_file`?

Reply:
It's still a file, even if it's been loaded into memory.